File: 24-05-2022/Min_sum_formed_by_digits.cpp
class Solution{
    public:
    long long int minSum(int arr[], int n)
    {
        // Your code goes here
        vector<long long> v,v1;
        sort(arr,arr+n);
        
        string s1="0",s2="0";
        for(long long i=0;i<n;)
        {
            s1+=to_string(arr[i]);
            i++;
            if(i<n)
            {
                s2+=to_string(arr[i]);
                i++;
            }
        }
        long long temp=stoll(s1)+stoll(s2);
        return temp;
    }
};

File: 24-05-2022/Min sum formed by digits.java
class Solution {
    
    public static long minSum(int arr[], int n)
    {
        // Your code goes here
        Arrays.sort(arr);
        
        String s1="0",s2="0";
        for(int i=0;i<n;)
        {
            s1+=Integer.toString(arr[i]);
            i++;
            if(i<n)
            {
                s2+=Integer.toString(arr[i]);
                i++;
            }
        }
        long temp=Long.parseLong(s1)+Long.parseLong(s2);
        return temp;
    }
}
